src/query/api/v1/handler/prometheus/debugger/sample_data.json

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
Why not add in a sample with 100,000 datapoints to pad lines of code 😛  ?

Reply:
Don't worry, I'm deleting this file :)

Reply:
Make sure to make it a separate commit to really get those numbers up ;)

Reply:
Of course